app/src/i18n/locales/en/data.json

```diff
@@ -287,5 +287,6 @@
   "file-deletion-error-description": "Something went wrong during file deletion",
   "file-deletion-success": "file deleted successfully",
 
-  "coming-soon": "Coming Soon"
+  "coming-soon": "Coming Soon",
+  "intlNumber": "{{val, number}}"
```

Comment:
We could move this to translation.json, so it is available on any page? Fine otherwise though.